.buildkite/ftr_security_serverless_configs.yml

```diff
@@ -12,6 +12,9 @@ disabled:
   - x-pack/test_serverless/functional/test_suites/security/cypress/security_config.ts
   - x-pack/test/security_solution_cypress/serverless_config.ts
 
+  # Playwright
+  - x-pack/test/security_solution_cypress/serverless_config.ts
```

Comment:
This according to the folder [here](https://github.com/elastic/kibana/pull/190803/files#diff-d2ff917f2c7d95a210f5212b6bf7f6d4ffaff1c38ccd32420169b6a11845e7a7R14) should be `x-pack/test/security_solution_playwright`. Correct?

Reply:
Right, this is a copy-paste mistake